### ISSUE TITLE
New catalog-app based on CKAN 2.8

### ISSUE THREAD

**Issue by avdata99** [GSA/datagov-ckan-multi#259]
## User Story

As a GSA developer, I want a new _catalog-app_ version based on CKAN 2.8 new using the OKF docker image so that I'm able to continue working in this CKAN upgrade.

## Acceptance Criteria
- [x] We have a catalog app running on CKAN 2.8 using the OKF docker image
- [ ] Catalog app does not contain any unused code
- [x] Tests are green
- [x] We have CI

## Task-list
- [x] Validate if using the OKF docker image is the right one (versus Viderum docker image)
- [x] Agree whether we do via a branch of catalog or a new repo [Aaron created a new repo]
- [x] Create a new branch for CKAN 2.8: [Done here](https://github.com/GSA/catalog-app/tree/catalog_ckan_2.8)
- [x] Add OKF CKAN docker. Do this by replacing the docker compose file (ideally we do this using the same one as OKF - we may need to config slightly, let’s see). Added in [this PR](https://github.com/GSA/catalog-app/pull/126) 
- [ ] Check that this is stable
   - [x] Update makefile: Covered in [this PR](https://github.com/GSA/catalog-app/pull/127/files)
   - [x] Tests are updated to bats file (as in the Invetory-app) and are green for the base CKAN image. Covered in [this PR](https://github.com/GSA/catalog-app/pull/129/files)
   - [x] CircleCI runs: tests are green [in CircleCI](https://circleci.com/gh/GSA/catalog-app/271?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)
   - [ ] Any changes or config required we do via raising issues and PR so we have a documented approach to coding
- [x] Send a PR for Adam to review
- [ ] Aaron to review
- [ ] Share with team (Julie, Paul etc)
- [ ] What do we want to improve in the catalog app (remove unnecessary/redundant code, hacks etc) - need to create a new issue for this "Improve catalog app"
- [ ] Not necessarily the last step, but we also need to involve Tom on this re security documentation

Next steps captured in #[264](https://github.com/GSA/datagov-ckan-multi/issues/264)

## Details

Esteban: Discussed with Paul Walsh the difference between the OKF version and Viderum version of the docker image. The Viderum image is more customizable (mainly build that way to work with CCO) but the OKF version is seen as the simplest, best docker image out there. Developed by Adria over a number of years and it works really well.

**Comment by kimwdavidson** [comment]
Blocked by analysis of #259

**Comment by kimwdavidson** [comment]
(closed follow up ticket #264 until work is better defined)

**Comment by kimwdavidson** [comment]
No longer needed. Taking new approach.
